server/src/org/broadinstitute/reaper/server/Reaper.scala

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.reaper.server
+
+import pureconfig.error.ConfigReaderFailures
+
+object ReaperRoutes extends cask.Routes {
```

Comment:
Could this be a `class` taking a `config: ReaperConfig`, to avoid the static `Reaper.config` reference in the version method? Not a big deal for this, but would be good to set the pattern so we don't walk back into Clio's setup where it's hard to mock out config values in tests.